### ISSUE TITLE
spy on inner method call failed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dlsgusrn7577/mockito#90]
```
What steps will reproduce the problem?
look into the testcase

What is the expected output? What do you see instead?
i expected "b return value" but see instead null

What version of the product are you using? On what operating system?
Mokito 1.7 on Windows XP SP3

```

Original issue reported on code.google.com by `jens.thi...@gmail.com` on 9 Jun 2009 at 2:22

Attachments:
- [SpyTest.java](https://storage.googleapis.com/google-code-attachments/mockito/issue-90/comment-0/SpyTest.java)

**Comment by GoogleCodeExporter** [comment]
```
I'll make you a happy man.

I'm just releasing 1.8.0-rc1. One of the improvements was making spy() a true 
partial
mock. Fixed in trunk/1.8.0-rc1.
```

Original comment by `szcze...@gmail.com` on 9 Jun 2009 at 2:59
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `szcze...@gmail.com` on 9 Jun 2009 at 3:15
- Added labels: **Milestone-Release1.8**
